test/integration/test_helper.exs

```diff
@@ -0,0 +1,6 @@
+ExUnit.start(capture_log: true)
+
+HTTPoison.start()
+
+defmodule TestHelper do
```

Comment:
Why do we needs this here?

Reply:
Why do we need what?

Reply:
This module. It's empty anyway.

Reply:
Not to mention, that putting it here was a mistake from beginning - scripts should never declare modules, unless they itself need it.